tags/resources/tag-formation-templates/template-leadership-election-process.md

```diff
@@ -1,3 +1,87 @@
+
+The `<TAG NAME>` does not have a current limit on the amount of leadership roles. <!-- ! UPDATE IF DISCUSSED OTHERWISE ! -->
+
+Final nominations will adhere to the [CNCF's TOC described process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections) as well as to the leadership nomination requirements described below.
+
+Any individual who is a member of the `<TAG NAME>`, its working groups, or otherwise considered an active participant of the group may nominate another individual. <!-- Update if your TAG has a more restricted pool of nominators -->
+
+The process of endorsing and selecting individuals will consider the following factors to ensure both the need for proper community representation and the necessary technical expertise: <!-- If the TAG has any additional items, such as leading a minimum number of projects, or facilitating a number meetings, they should be added to this list under "Body of work" -->
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Number of nominations from different companies, people, regions for a single individual
+* Existing body of work
+* Justification provided during nomination
+
+In case there are no individuals expressing interest in taking up leadership positions, an evaluation process for the TAG / working group or project archival may be initiated, as [outlined by the TOC](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#retirement).
+
+### Community timeline
+
+Community nominations will run for at least 1 month, with a maximum up to 2 months.
+
+* For TAG Chair nominations, the TOC will take a vote as [described by the TOC](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+* For TAG TL, TAG Chairs will vote and propose the nomination to the TOC. The TOC takes the final vote.
+* For Working Group and Project Chairs, the TAG Chairs and Leads will take a vote.
+
+Upon close of nominations, a decision should be made within the next month.
+
+### Submitting a nomination
+
+Nominations are to be submitted to [TAG Co-Chair's email alias](mailto:cncf-tag-chairs@lists.cncf.io) with the details: <!-- ! UPDATE EMAIL with the TAG's Leadership's correct mailing list ! -->
+
+* Subject "NOMINATION $(TAG Chair, TAG TL, WG XYZ Chair): $NAME-OF-NOMINEE"
+* CC the nominee
+* Justification
+* Nominee Biography
+
+## Nomination requirements
+
+The `<TAG NAME>` recognizes and values the achievements of individuals who have proven themselves in this community.
+If a given nominee does not meet all the requirements, the TAG leadership will still review the nomination and justification to determine if enough intent and commitment is present to move forward with an endorsement.
+
+**Nominees must**:
+
+* Be the author of at least one pull request against the `<TAG NAME>` repository
```

Comment:
A quick question:  What is the purpose of calling this out versus making it part of the list below?   It seems to already be somewhat related to the commenting on PRs, etc. issue below.

Is the author of a PR the person who created it?   Is any person who did a lot of the work on it an author?   What about someone who reviewed and commented heavily on it?   Do all PRs regardless of length / purpose / complexity count the same here?   

I know this is all subjective and I'm not proposing to define all the corner cases.   I'd actually prefer to make this less specific and less emphasized.   My concern is that I don't want someone with imposter syndrome who has been a great contributor, but who didn't create a PR to feel like their application shouldn't be put forth.